launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css

```diff
@@ -0,0 +1,60 @@
+.center-page {
+    left: 50%;
```

Comment:
Style nit:  Can we either put attributes that start with "-" either all at the end of the block or at the top and always keep it that way?

Reply:
done.